tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -95,15 +94,17 @@ export async function prepareProjectForE2e(name: string) {
 }
 
 export function useBuiltPackagesVersions(): Promise<void> {
+  const packages = getGlobalVariable('package-tars');
+
   return updateJsonFile('package.json', (json) => {
     json['dependencies'] ??= {};
     json['devDependencies'] ??= {};
 
     for (const packageName of Object.keys(packages)) {
       if (packageName in json['dependencies']) {
-        json['dependencies'][packageName] = packages[packageName].version;
+        json['dependencies'][packageName] = packages[packageName];
```

Comment:
Since we have access to the version in `findPackageTars` can we keep using the version instead?